atomicapp/nulecule/container.py

```diff
@@ -90,7 +90,7 @@ def pull(self, image, update=False):
 
         cockpit_logger.info('Skipping pulling docker image: %s' % image)
 
-    def extract(self, image, source, dest, update=False):
+    def extract(self, image, source, dest, update=False, file=False):
```

Comment:
@cdrage - here is a 5 minute try at this: http://paste.fedoraproject.org/389012/46799478

I would suggest we do this in a separate PR and then rebase this PR on top.